documentation/airport-format.md

```diff
@@ -304,13 +304,21 @@ They're used when we need aircraft to fly over a location that doesn't have an a
             ["N36d4m34.82", "W115d10m16.98", "2179ft"],
             ["N36d4m35.05", "W115d7m15.93", "2033ft"]
         ],
-        "ils": [false, true]
+        "ils": [false, true],
+        "ils_distance":[30, 25],
+        "loc_maxDist": [28, 20],
+        "glideslope": [3.00, 2.50]
     }
 ],
 ```
 - **name** - Name of each runway in the pair.  Names should reflect a 180 degree difference. so if one end if `"Runway 9"` (or `"Runway 09"`, depending on the country) the other runway should be `"Runway 27"`.
 - **end** - Latitude, Longitude, and Elevation of the runway threshold (the spot where the numbers would be painted). _see [lat, lon, elev](#latitude-longitude-elevation) for formatting_
 - **ils** - Boolean property used to indicate if a runway has an ILS approach
+- **ils_distance** - Distance the ILS extends away from the runway
+- **glideslope** - Descent angle of the ILS glideslope
+- **loc_maxDist** -
+- **ils_gs_maxHeight** - _not found in any airport file_
+- **sepFromAdjacent** - _not found in any airport file_
```

Comment:
Maximum distance from the runway threshold where the localizer is still usable by aircraft, in nm.

Maximum height where the glideslope is still usable by aircraft, in ft MSL.

A way to manually specify the separation required between this runway and an adjacent runway, in nm. This will most likely be deprecated in favor of the plans from #37.